app/design/frontend/base/default/layout/ebanx.xml

```diff
@@ -176,7 +188,10 @@
             <action method="addJs">
                 <script>ebanx/vanilla-masker/vanilla-masker.min.js</script>
             </action>
-
+            <action method="addItem">
+                <type>skin_js</type>
+                <script>ebanx/js/input-handler.js</script>
```

Comment:
Input handlers are important, but we don't need it twice

Reply:
In this case, has 2 because we need the input-handler file inside, edit.html and inside the checkout, and to do this I need import the script twice